src/backend/utils.ts

```diff
@@ -969,14 +1005,47 @@ export async function downloadDefaultWine() {
   return null
 }
 
+export async function setGPTKDefaultOnMacOS() {
```

Comment:
If compatible with GPTK we also want to setup a new prefix as default to avoid issues with DXVK installed on a previous Wine prefix.
The downside is for games that stores saves locally, they will be 'lost' so will need to be moved from the previous prefix to a new one.
But this is a global config, not individual, so it is unlikely to happen.